### DISCUSSION TITLE
Curious about the use of 'collection' terminology

### DISCUSSION THREAD

**Discussion by alea12** [pocketbase/pocketbase#3842] [Q&A] [answered]
Hello! I've recently come across this project and I'm quite intrigued by the idea of a lightweight, portable alternative to Firebase.

I couldn't help but notice the use of the term `collection`, which is commonly associated with NoSQL (document-oriented) databases like Firestore, while PocketBase is built on SQLite. Could someone explain the choice of this terminology within the context of PocketBase, which is based on a SQL database?

**Comment by ganigeorgiev** [comment, marked as answer]
It is just a word, don't overthink it. A `collection` in PocketBase context is a collection/list/set of records (usually db table entries).

`collection` was chosen primarily to make it more clear that it is not just SQLite table. A PocketBase `collection` could define fields schema, crud API rules, fields validation or even depending on its type (`base`, `auth`, `view`) different API behavior.

**Reply by alea12** [reply]
Got it, thank you! Was just curious 🙂
